<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewForecast</class>
 <widget class="QWidget" name="ViewForecast">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>531</width>
     <height>401</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="accessibleName">
     <string comment="График"/>
    </property>
    <attribute name="title">
     <string>График</string>
    </attribute>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>500</width>
       <height>350</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>350</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Таблица</string>
    </attribute>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>391</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>330</y>
       <width>85</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonDeleteRow">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>71</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonAddRow">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>9</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonPasteRow">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>40</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Вставить</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Настройка нейронной сети</string>
    </attribute>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>500</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>330</y>
       <width>251</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Сформировать нейронную сеть</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать нейронную сеть</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>71</y>
     <width>198</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Доступные нейронные сети</string>
   </property>
  </widget>
  <widget class="QComboBox" name="NeuralNetwork">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>100</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>72</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип сырья</string>
   </property>
  </widget>
  <widget class="QComboBox" name="TypeMaterial">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>40</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>46</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Марка</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Mark">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
